### ISSUE TITLE
[armhf] repos missing

### ISSUE THREAD

**Issue by meyskens** [multiarch/debian-debootstrap#1]
The Debian repos are missing from the image, only the security one is there.

```
root@9c0651a4415f:/# cat /etc/apt/sources.list
deb http://security.debian.org jessie/updates main
```

**Comment by moul** [comment]
Hi @meyskens 

This looks not regular:

``` console
$ for distro in jessie sid; do for arch in amd64 i386 amd64 armhf; do echo docker run -it --rm multiarch/debian-debootstrap:$arch-$distro cat /etc/apt/sources.list; docker run -it --rm multiarch/debian-debootstrap:$arch-$distro cat /etc/apt/sources.list | sed 's/^/- /' 2>/dev/null; done; done
docker run -it --rm multiarch/debian-debootstrap:amd64-jessie cat /etc/apt/sources.list
- deb http://ftp.us.debian.org/debian jessie main
- deb http://ftp.us.debian.org/debian jessie-updates main
- deb http://security.debian.org jessie/updates main
docker run -it --rm multiarch/debian-debootstrap:i386-jessie cat /etc/apt/sources.list
- deb http://ftp.us.debian.org/debian jessie main
- deb http://ftp.us.debian.org/debian jessie-updates main
- deb http://security.debian.org jessie/updates main
docker run -it --rm multiarch/debian-debootstrap:amd64-jessie cat /etc/apt/sources.list
- deb http://ftp.us.debian.org/debian jessie main
- deb http://ftp.us.debian.org/debian jessie-updates main
- deb http://security.debian.org jessie/updates main
docker run -it --rm multiarch/debian-debootstrap:armhf-jessie cat /etc/apt/sources.list
- deb http://security.debian.org jessie/updates main
docker run -it --rm multiarch/debian-debootstrap:amd64-sid cat /etc/apt/sources.list
- deb http://ftp.us.debian.org/debian sid main
docker run -it --rm multiarch/debian-debootstrap:i386-sid cat /etc/apt/sources.list
- deb http://ftp.us.debian.org/debian sid main
docker run -it --rm multiarch/debian-debootstrap:amd64-sid cat /etc/apt/sources.list
- deb http://ftp.us.debian.org/debian sid main
docker run -it --rm multiarch/debian-debootstrap:armhf-sid cat /etc/apt/sources.list
```

---

all those images are generated using debootstrap
if we can't rely on debootstrap, maybe we should hardcode the sources.list file from the generator

what do you think ?

**Comment by meyskens** [comment]
Probably a good idea, Debian uses the same mirrors for all archs.
These are the official ones from Debian's Docker image using httpredir.debian.org

```
deb http://httpredir.debian.org/debian jessie main
deb http://httpredir.debian.org/debian jessie-updates main
deb http://security.debian.org jessie/updates main
```

**Comment by rsebille** [comment]
Hi,

FYI, since the today update (2018-05-25T13:01:33.625952226Z), this problem is back :(.

```console
$ for distro in jessie stretch; do for arch in i386 amd64 armhf; do echo docker run -it --rm multiarch/debian-debootstrap:$arch-$distro cat /etc/apt/sources.list; docker run -it --rm multiarch/debian-debootstrap:$arch-$distro cat /etc/apt/sources.list | sed 's/^/- /' 2>/dev/null; done; done
docker run -it --rm multiarch/debian-debootstrap:i386-jessie cat /etc/apt/sources.list
- deb http://deb.debian.org/debian jessie main
- deb http://deb.debian.org/debian jessie-updates main
- deb http://security.debian.org jessie/updates main
docker run -it --rm multiarch/debian-debootstrap:amd64-jessie cat /etc/apt/sources.list
- deb http://deb.debian.org/debian jessie main
- deb http://deb.debian.org/debian jessie-updates main
- deb http://security.debian.org jessie/updates main
docker run -it --rm multiarch/debian-debootstrap:armhf-jessie cat /etc/apt/sources.list
- deb http://security.debian.org jessie/updates main
docker run -it --rm multiarch/debian-debootstrap:i386-stretch cat /etc/apt/sources.list
- deb http://ftp.us.debian.org/debian stretch main
- deb http://ftp.us.debian.org/debian stretch-updates main
- deb http://security.debian.org stretch/updates main
docker run -it --rm multiarch/debian-debootstrap:amd64-stretch cat /etc/apt/sources.list
- deb http://ftp.us.debian.org/debian stretch main
- deb http://ftp.us.debian.org/debian stretch-updates main
- deb http://security.debian.org stretch/updates main
docker run -it --rm multiarch/debian-debootstrap:armhf-stretch cat /etc/apt/sources.list
- deb http://security.debian.org stretch/updates main

```

Edit: Fixed with the `2018-05-26T17:09:14.879555296Z` push, thanks! :)
